twill-yarn/src/main/java/org/apache/twill/internal/ServiceMain.java

```diff
@@ -264,6 +262,10 @@ protected String getLoggerLevel(Logger logger) {
     return "OFF";
   }
 
+  protected String getLogLevels() {
+    return "";
```

Comment:
For a launch, the `TwillContainerService` is constructed before the run of `doMain` method, which means the we have not called `configureLogger()` method at that time so log level setting at constructor will not work.